src/LibDeclarations/webxr.nativeextensions.d.ts

```diff
@@ -1,11 +1,82 @@
-// This file contains native only extensions for WebXR  These APIs are not supported in the browser yet.
+// This file contains native only extensions for WebXR. These APIs are not supported in the browser yet.
 // They are intended for use with either Babylon Native https://github.com/BabylonJS/BabylonNative or
 // Babylon React Native: https://github.com/BabylonJS/BabylonReactNative
 
+// Open question: Do better types exist to use for vectors/quaternions?

```

Comment:
Open question: Do better types exist to use for vectors/quaternions?

Reply:
DOMPoint (https://drafts.fxtf.org/geometry-1/#DOMPoint) can be used for both types (and is being used extensively in XR). And you can use DOMPointReadOnly when it makes sense
Or https://immersive-web.github.io/webxr/#xrrigidtransform directly, if it makes more sense?